firmware/pattern.c

```diff
@@ -12,7 +12,7 @@
 #include "uptime.h"
 #include "version_data.h"
 
-#define PATTERN_FRAMEBUFFER_BASE 0x02000000 + 0x100000
+#define PATTERN_FRAMEBUFFER_BASE 0x00000000 + 0x100000
```

Comment:
Pattern is now the first framebuffer? Can you add a comment about this?

Reply:
Not sure what sort of comment you want. The commit message making this change describes it.
What is special enough about the first frame buffer that warrants a comment?

Reply:
Maybe, 
```c
/* We store all the frame buffers at offset 0x0000000, the layout can be seen below;
 *
 *  0xX00000 - Pattern Frame Buffer
 * Each input then has 3 frame buffers spaced like this;
 *  0xX00000 - HDMI Input 0 - Frame Buffer n
 *  0xX00000 - HDMI Input 0 - Frame Buffer n+1
 *  0xX00000 - HDMI Input 0 - Frame Buffer n+2
```